src/server.c

```diff
@@ -416,7 +344,14 @@ main(int argc, char **argv) {
     lwsl_notice("tty configuration:\n");
     if (server->credential != NULL)
         lwsl_notice("  credential: %s\n", server->credential);
-    lwsl_notice("  start command: %s\n", server->command);
+    
+    lwsl_notice("  start command: \n");
+    for (c = 0; c < argc; c++) {
+        lwsl_notice(" %s\n", argv[c]);
```

Comment:
I mean the start command, for example, `ttyd bash -x` outputs:

```
[2017/08/12 23:22:36:5915] NOTICE: ttyd 1.3.3 (libwebsockets 2.2.0)
[2017/08/12 23:22:36:5918] NOTICE: tty configuration:
[2017/08/12 23:22:36:5918] NOTICE:   start command:
[2017/08/12 23:22:36:5918] NOTICE:  bash
[2017/08/12 23:22:36:5918] NOTICE:  -x
[2017/08/12 23:22:36:5918] NOTICE:
[2017/08/12 23:22:36:5918] NOTICE:   reconnect timeout: 10s
```
What I expect is (behaves the same as before):

```
[2017/08/12 23:22:36:5915] NOTICE: ttyd 1.3.3 (libwebsockets 2.2.0)
[2017/08/12 23:22:36:5918] NOTICE: tty configuration:
[2017/08/12 23:22:36:5918] NOTICE:   start command: bash -x
[2017/08/12 23:22:36:5918] NOTICE:   reconnect timeout: 10s
```